src/tests/integration/D3MCompound.t.sol

```diff
@@ -69,6 +71,9 @@ interface InterestRateModelLike {
 }
 
 contract D3MCompoundTest is DSSTest {
+    Hevm hevm;
```

Comment:
We have deprecated `dapp` and `hevm`, you should use directly `vm` (without declaring it) and cheat code is also not necessary.

Reply:
right, done